build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -1,4 +1,6 @@
 parameters:
+  BuildForHelix: 'true'
```

Comment:
FWIW, I'm not sure if this is really needed/used. But since this is copied from the internal repo, it's probably fine to leave it as-is for now.

Reply:
Consistency hath its virtues :-)

There's other improvements we should make to the pipeline, but they're unrelated to this. Thoughts for the future